tests/functional/logic_test.sh

```diff
@@ -4,6 +4,50 @@ ROOT_DIR="$(dirname "${BASH_SOURCE[0]}")"
 
 SCRIPT="$ROOT_DIR/logic.sh"
 
-function test_your_logic() {
-  assertEquals "expected 123" "$($SCRIPT "123")" "Functional test working!"
```

Comment:
I would keep here only the `test_your_logic` as how it was before, because you already moved the tests to the example dir

Reply:
What do you think @khru @Tito-Kati ?